HTML-Forms: Mint Signup Page

DeliceLydia opened the pull request:
In this project, we created a replica of mint.com signup page using HTML and CSS.

Additional Information:
- Setup project dependencies like GitHub Actions and Linters via Microverse Linters Config.
- Defined a well structured DOM with each element properly grouped together using proper sizes and alignments.
- Used the browser's console to review mint.com's development strategy.
- Used the right input tag and attributes in the form.

eananti committed:
Update README file

Added project description
Added project technologies
Added project authors

DeliceLydia committed:
Initial commit

eananti committed:
Restructure HTML

eananti committed:
Update form

Added fieldset
Add id and for attributes

eananti committed:
Define base CSS style

eananti committed:
Add layout to body elements

eananti committed:
Add border to main

DeliceLydia committed:
Add html signup page(signup):
add header elements
add main elements
add footer elements

DeliceLydia committed:
Add width to main elements

eananti committed:
Redesign the top header section

Used grid to reposition top header
Vertically centered the paragraph container

eananti committed:
Add dependencies link to README

eananti committed:
Fixed linter css errors

eananti committed:
Add layout to main and footer

Added padding to main contents
Added margin to footer container

eananti committed:
Float list items

eananti committed:
Fix top header overflow

eananti committed:
Add white packlock to buttons

eananti committed:
Add fonts to footer element

eananti committed:
Style input field and submit button

eananti committed:
Add button icon

eananti committed:
Fix linter specificity error

eananti committed:
Add website screenshot

SpaYco commented on index.html:
```diff
@@ -0,0 +1,78 @@
+                </button>
+            </div>
+        </div>
+        <div class="logo center-align">
+            <img src="./images/logo.png" alt="intuit logo">
+        </div>
```
- [x] [REQUIRED] the Intuit products should be in images and text, not a screenshot, please fix that

SpaYco commented on index.html:
```diff
@@ -0,0 +1,78 @@
+                    <img src="./images/lock_open.png" alt="padlock">
+                    <input type="password" name="password" id="password" required>
+                </fieldset>
+                <button class="form-button" type="button" name="submit">
+                    <img src="./images/lock.png" alt="sign in"> 
+                    <span>Create Account</span>
+                </button>
```
- [x] the button should have a `submit` type, not `button`  ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/html_css/html_forms/form_and_input_requirement.png) from [Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/59524927/))

SpaYco reviewed (changes requested):
# Changes Requested 

Hello Team! 👋 
Awesome job so far!! 👍 you still need to make some changes though! 

## Required Changes ♻️ 

- [x] your PR description can be more descriptive, please change it.

- I included more required changes below, please check them out

## Optional

- [x] make the email and password inputs required


### please make the appropriate changes and ask for another code review ✔️ 

Happy Coding!

Best, 
Aziz Mejri 
_contact me on [twitter](https://twitter.com/iSpaYco) or slack (@SpaYco) if you need any help_

DeliceLydia committed:
Fix npx errors

DeliceLydia committed:
Fix top header padding issues

eananti committed:
Fix issues from review
Used images for logo and products
Added submit type to buttons
Added fixed privacy button

DeliceLydia committed:
Change font size
p elements
links

eananti committed:
Remove comment

Deleted commented property on css

DeliceLydia committed:
Add line-height p elements

DeliceLydia committed:
Fix linter errors

eananti committed:
Add box shadow to input

eananti committed:
Fix linter error

andresporras3423 reviewed (approved):
# Status: Approved :heavy_check_mark: :heavy_check_mark: :heavy_check_mark:
**Hello @DeliceLydia @eananti** 
Amazing job :tada: :tada: :tada:  you have met all the requirements for this project
Now you are ready to continue with your next challenge :muscle: :muscle:
Don't forget to merge with the master branch

Keep going like this  :clap: :clap:

Oscar Andrés Russi Porras
